src/main/java/com/skypro/calculator/rest/EmployeeController.java

```diff
@@ -0,0 +1,51 @@
+package com.skypro.calculator.rest;
+
+import com.skypro.calculator.model.Employee;
+import com.skypro.calculator.services.EmployeeCollectionsService;
+import com.skypro.calculator.services.EmployeeService;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.Set;
+
+@RestController
+@RequestMapping("/employee")
+public class EmployeeController {
+
+    private EmployeeCollectionsService employeeService;
+
+    public EmployeeController(@Qualifier("employeeServiceCollectionsImpl") EmployeeCollectionsService employeeService) { this.employeeService = employeeService;  }
+
+    @GetMapping("/")
+    public String greetEmployee(){
+        return "Hello employees!";
+    }
+
+    @GetMapping("/add")
+    public Employee addEmployee(@RequestParam("firstName") String firstName,
```

Comment:
если имена параметров совпадают, то строку в аннотации можно опустить
```suggestion
    public Employee addEmployee(@RequestParam String firstName,
```